src/averageAges.js

```diff
@@ -14,12 +14,16 @@
  *
  * @return {number}
  */
+
+const calculateAverageAge = (people) => people.reduce(
+  (sum, person) => sum + person.died - person.born, 0) / people.length;
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => century
```

Comment:
Consider breaking down the ternary operator into separate lines for better readability.